hbase-server/src/main/java/org/apache/hadoop/hbase/master/normalizer/SimpleRegionNormalizer.java

```diff
@@ -284,7 +284,7 @@ private double getAverageRegionSizeMb(final List<RegionInfo> tableRegions,
     double avgRegionSize;
     int targetRegionCount = tableDescriptor.getNormalizerTargetRegionCount();
     long targetRegionSize = tableDescriptor.getNormalizerTargetRegionSize();
-    LOG.debug("Table {} configured with target region count {}, target region size {}", table,
+    LOG.debug("Table {} configured with target region count {}, target region size {} MB", table,
```

Comment:
For sure these are MB values? I tried looking at code but it doesn't say explicitly...

Reply:
Oh, there are some checkstyle complaints to fix?

Reply:
>For sure these are MB values? I tried looking at code but it doesn't say explicitly...

Yeah, but there is a hint of it in the [alter](https://github.com/apache/hbase/blob/5457554826519232c7d81ab382a5f345da0c0da2/hbase-shell/src/main/ruby/shell/commands/alter.rb#L51) command of hbase shell. 

Also, `avgRegionSize` is set to `targetRegionSize` if its set, else the computed value for `avgRegionSize` is  in MBs. So its an inference.